wandb: Currently logged in as: nashokkumar (ml4ed). Use `wandb login --relogin` to force relogin
wandb: wandb version 0.13.4 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.13.3
wandb: Run data is saved locally in /work/nashokkumar_umass_edu/nipschal/Neurips-Challenge-22/wandb/run-20221012_034425-3uu1ufvs
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run nips_embed_300_mask
wandb: ⭐️ View project at https://wandb.ai/ml4ed/predict-graph
wandb: 🚀 View run at https://wandb.ai/ml4ed/predict-graph/runs/3uu1ufvs
/work/nashokkumar_umass_edu/nipschal/Neurips-Challenge-22/predict_graph_embed_mask.py:541: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  labels = torch.tensor(dataset_tensor[:, 1, :].clone().detach(), dtype=torch.long)
/work/nashokkumar_umass_edu/.conda/envs/nistorch/lib/python3.10/site-packages/torch/nn/parallel/_functions.py:68: UserWarning: Was asked to gather along dimension 0, but all input tensors were scalars; will instead unsqueeze and return a vector.
  warnings.warn('Was asked to gather along dimension 0, but all '
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [70,0,0], thread: [27,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [70,0,0], thread: [28,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [70,0,0], thread: [29,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [70,0,0], thread: [30,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [70,0,0], thread: [31,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [27,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [28,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [29,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [30,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [31,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [59,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [60,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [61,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [62,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [63,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [91,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [92,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [93,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [94,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [95,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [123,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [124,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [125,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [126,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [52,0,0], thread: [127,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [91,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [92,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [93,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [94,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [95,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [59,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [60,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [61,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [62,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [63,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [123,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [124,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [125,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [126,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [127,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [27,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [28,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [29,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [30,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [71,0,0], thread: [31,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [59,0,0], thread: [59,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [59,0,0], thread: [60,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [59,0,0], thread: [61,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [59,0,0], thread: [62,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [59,0,0], thread: [63,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [59,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [60,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [61,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [62,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [63,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [91,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [92,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [93,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [94,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [95,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [123,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [124,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [125,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [126,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [127,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [27,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [28,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [29,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [30,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [66,0,0], thread: [31,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [27,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [28,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [29,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [30,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [31,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [59,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [60,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [61,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [62,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [63,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [91,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [92,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [93,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [94,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [95,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [123,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [124,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [125,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [126,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484772832/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:145: operator(): block: [48,0,0], thread: [127,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
Traceback (most recent call last):
  File "/work/nashokkumar_umass_edu/nipschal/Neurips-Challenge-22/predict_graph_embed_mask.py", line 605, in <module>
    main()
  File "/work/nashokkumar_umass_edu/nipschal/Neurips-Challenge-22/predict_graph_embed_mask.py", line 590, in main
    model, epoch_train_loss, epoch_val_loss = train(epochs, dkt_model, train_dataloader, val_dataloader, optimizer, scheduler) # add val_dataloader later
  File "/work/nashokkumar_umass_edu/nipschal/Neurips-Challenge-22/predict_graph_embed_mask.py", line 489, in train
    valloss = model(b_input_ids_val, b_labels_val, epoch_i+1)
  File "/work/nashokkumar_umass_edu/.conda/envs/nistorch/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1130, in _call_impl
    return forward_call(*input, **kwargs)
  File "/work/nashokkumar_umass_edu/.conda/envs/nistorch/lib/python3.10/site-packages/torch/nn/parallel/data_parallel.py", line 168, in forward
    outputs = self.parallel_apply(replicas, inputs, kwargs)
  File "/work/nashokkumar_umass_edu/.conda/envs/nistorch/lib/python3.10/site-packages/torch/nn/parallel/data_parallel.py", line 178, in parallel_apply
    return parallel_apply(replicas, inputs, kwargs, self.device_ids[:len(replicas)])
  File "/work/nashokkumar_umass_edu/.conda/envs/nistorch/lib/python3.10/site-packages/torch/nn/parallel/parallel_apply.py", line 86, in parallel_apply
    output.reraise()
  File "/work/nashokkumar_umass_edu/.conda/envs/nistorch/lib/python3.10/site-packages/torch/_utils.py", line 461, in reraise
    raise exception
RuntimeError: Caught RuntimeError in replica 0 on device 0.
Original Traceback (most recent call last):
  File "/work/nashokkumar_umass_edu/.conda/envs/nistorch/lib/python3.10/site-packages/torch/nn/parallel/parallel_apply.py", line 61, in _worker
    output = module(*input, **kwargs)
  File "/work/nashokkumar_umass_edu/.conda/envs/nistorch/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1130, in _call_impl
    return forward_call(*input, **kwargs)
  File "/work/nashokkumar_umass_edu/nipschal/Neurips-Challenge-22/predict_graph_embed_mask.py", line 312, in forward
    hidden_states = self.gru(input_embed, epoch) # initially hidden_states, _
  File "/work/nashokkumar_umass_edu/.conda/envs/nistorch/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1130, in _call_impl
    return forward_call(*input, **kwargs)
  File "/work/nashokkumar_umass_edu/nipschal/Neurips-Challenge-22/predict_graph_embed_mask.py", line 176, in forward
    lower = self.permuted_matrix(epoch, verbose=False) # (PLP')'
  File "/work/nashokkumar_umass_edu/.conda/envs/nistorch/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1130, in _call_impl
    return forward_call(*input, **kwargs)
  File "/work/nashokkumar_umass_edu/nipschal/Neurips-Challenge-22/predict_graph_embed_mask.py", line 91, in forward
    print(self.matrix)
  File "/work/nashokkumar_umass_edu/.conda/envs/nistorch/lib/python3.10/site-packages/torch/nn/parameter.py", line 61, in __repr__
    return 'Parameter containing:\n' + super(Parameter, self).__repr__()
  File "/work/nashokkumar_umass_edu/.conda/envs/nistorch/lib/python3.10/site-packages/torch/_tensor.py", line 338, in __repr__
    return torch._tensor_str._str(self, tensor_contents=tensor_contents)
  File "/work/nashokkumar_umass_edu/.conda/envs/nistorch/lib/python3.10/site-packages/torch/_tensor_str.py", line 481, in _str
    return _str_intern(self, tensor_contents=tensor_contents)
  File "/work/nashokkumar_umass_edu/.conda/envs/nistorch/lib/python3.10/site-packages/torch/_tensor_str.py", line 447, in _str_intern
    tensor_str = _tensor_str(self, indent)
  File "/work/nashokkumar_umass_edu/.conda/envs/nistorch/lib/python3.10/site-packages/torch/_tensor_str.py", line 270, in _tensor_str
    formatter = _Formatter(get_summarized_data(self) if summarize else self)
  File "/work/nashokkumar_umass_edu/.conda/envs/nistorch/lib/python3.10/site-packages/torch/_tensor_str.py", line 103, in __init__
    nonzero_finite_vals = torch.masked_select(tensor_view, torch.isfinite(tensor_view) & tensor_view.ne(0))
RuntimeError: CUDA error: device-side assert triggered
CUDA kernel errors might be asynchronously reported at some other API call,so the stacktrace below might be incorrect.
For debugging consider passing CUDA_LAUNCH_BLOCKING=1.

wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: - 0.001 MB of 0.001 MB uploaded (0.000 MB deduped)wandb: \ 0.001 MB of 0.001 MB uploaded (0.000 MB deduped)wandb: | 0.001 MB of 0.001 MB uploaded (0.000 MB deduped)wandb: / 0.001 MB of 0.625 MB uploaded (0.000 MB deduped)wandb: - 0.001 MB of 0.625 MB uploaded (0.000 MB deduped)wandb: \ 0.112 MB of 0.625 MB uploaded (0.000 MB deduped)wandb: | 0.625 MB of 0.625 MB uploaded (0.000 MB deduped)wandb: / 0.625 MB of 0.625 MB uploaded (0.000 MB deduped)wandb: - 0.625 MB of 0.625 MB uploaded (0.000 MB deduped)wandb: \ 0.625 MB of 0.625 MB uploaded (0.000 MB deduped)wandb: | 0.625 MB of 0.625 MB uploaded (0.000 MB deduped)wandb: / 0.625 MB of 0.625 MB uploaded (0.000 MB deduped)wandb:                                                                                
wandb: Synced nips_embed_300_mask: https://wandb.ai/ml4ed/predict-graph/runs/3uu1ufvs
wandb: Synced 6 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20221012_034425-3uu1ufvs/logs
